relm-derive/src/gen/mod.rs

```diff
@@ -28,7 +28,7 @@
 pub(crate) mod parser;
 
 mod adder;
-mod gen;
+mod generator;
```

Comment:
Was there a specific reason to rename the module?

Reply:
https://rust-lang.github.io/rust-clippy/master/index.html#module_inception

Reply:
:+1: I guess we can just use `generator` until I find better names ;) .